### ISSUE TITLE
Request: FM-Op: Add CV control of ratio

### ISSUE THREAD

**Issue by Undistraction** [bogaudio/BogaudioModules#162]
Would be great to be able to control the ratio with a CV. I've played around modulating the the v/Oct by multiplying by different values, but the result doesn't seem to be as nice or consistent as using the ratio control. Exposing control of the ratio via CV would allow some lovely movement between fixed ratios.

**Comment by mdemanett** [comment]
There's no room for more ports on FM-OP.  This might work on some future expanded version of FM-OP...I'll leave this open as a feature request.

**Comment by Undistraction** [comment]
@mdemanett Yep. I understand. Out of interest, would it be simple to replicate the effect of the ratio control (assuming the control was left at 1:1) by multiplying the incoming V/Oct by my desired ratio value, for example multiplying it by 2 to get a 2:1 ratio? Or is there more going on than that?

**Comment by mdemanett** [comment]
You'll have to respect the 1-volt-per-octave scaling to achieve that.  So to get a 2:1 ratio, you'd add 1V (add an octave, double the frequency).  Adding 2V will be a 4:1 ratio.  Subtracting 1V will be a 1:2, and so on.  A 3:1 is approximately +1.58V.

Module [STACK](https://github.com/bogaudio/BogaudioModules#-stack) will add a v/oct offset to an input v/oct according to musical intervals you set, and has CV -- might do what you want.

**Comment by Undistraction** [comment]
@mdemanett thank you. That's really helpful.
